src/brokers/broker-details/components/yaml/Yaml.container.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC, Suspense } from 'react';
+import { Loading } from '@app/shared-components/Loading/Loading';
+import { ResourceYAMLEditor } from '@openshift-console/dynamic-plugin-sdk';
+import YAML from 'yaml';
+import { Alert, AlertVariant, Button } from '@patternfly/react-core';
+import { useTranslation } from '@app/i18n/i18n';
+import { useNavigate, useParams } from 'react-router-dom-v5-compat';
+
+type YamlViewProps = {
+  brokerCr: object;
+};
+
+const YamlContainer: FC<YamlViewProps> = ({ brokerCr }) => {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+  const { ns: namespace, name } = useParams<{ ns?: string; name?: string }>();
+  const onClickEditYaml = () => {
+    const currentPath = window.location.pathname;
+    navigate(
+      `/k8s/ns/${namespace}/edit-broker/${name}?returnUrl=${encodeURIComponent(
+        currentPath,
+      )}`,
+    );
+  };
+
+  return (
+    <>
+      <Alert
+        variant={AlertVariant.info}
+        isInline
+        title={t('This YAML view is in read-only mode.')}
+        className="pf-u-mt-md pf-u-ml-md pf-u-mr-md"
+      >
+        <Button variant="link" onClick={onClickEditYaml}>
+          {t('Click here to update the deployment configuration in the YAML.')}
+        </Button>
```

Comment:
I think we should change this text, I've seen in an internal document that the usage of "click here" is very outdated. I grew up in the 90s so it seems fine to me, but apparently it is not. What about we update the test to:
`Proceed to the edit form to apply changes` and only set the `edit form` part of the sentence as the link to get to the edit form. Would that work for you?